backend/koofr/koofr.go

```diff
@@ -40,8 +40,11 @@ func init() {
 				Value: "koofr",
 				Help:  "Koofr, https://app.koofr.net/",
 			}, {
-				Value: "digistorage",
-				Help:  "Digi Storage, https://storage.rcs-rds.ro/",
+				Value: "digistoragero",
```

Comment:
I htink renaming this from `digistorage` to ``digistoragero` will break existing configs unless we have a workaround.

So either don't rename it, or put a workaround to recognise `digistorage` and `digistoragero` to be the same thing.

Reply:
Yeah, makes sense, I'll change it so that `digistorage` is an alias for `digistoragero`.